build_and_setup_everything_bazel_manually.sh

```diff
@@ -1,29 +1,46 @@
-#!/bin/sh
+#!/bin/bash
 
-[[ -z "${REMOTE_DOCKER_HOST}" ]] || . ./setup_remote_docker_kind.sh
+set -e
+
+source ./cluster/common.sh 
+[[ -z "${REMOTE_DOCKER_HOST}" ]] || . ./cluster/kind/setup_remote_docker_kind.sh
 
 
 [[ -z "${REMOTE_DOCKER_HOST}" ]] || setup_remote_docker
 
-. ./kind_with_registry.sh
+# Create kind  cluster with local docker registry
+. ./cluster/kind/kind_with_registry.sh
 
-./get_forklift_bazel.sh
+# clone the forklift repo
+./cluster/get_forklift_bazel.sh
 
-./k8s-deploy-kubevirt.sh
+# deploy kubevirt
+./cluster/k8s-deploy-kubevirt.sh
 
-./k8s-deploy-cert-manager.sh
+# deploy cert-manager
+./cluster/k8s-deploy-cert-manager.sh
 
+# build forklift from source and push to local docker registry
 ./build_forklift_bazel.sh
 
-./deploy_local_forklift_bazel.sh
+# deploy forklift from local docker registry
+./cluster/deploy_local_forklift_bazel.sh
+
+# run VMware provider setup
+./cluster/providers/vmware/setup.sh
+
+# run ovirt provider setup
+./cluster/providers/ovirt/setup.sh
 
-./vmware/setup.sh
+# run openstack/packstack provider setup
+./cluster/providers/openstack/setup.sh
 
-./ovirt/setup.sh
+# grant admin rights so its token can be used to access the API
+k8s_grant_permissions
 
-./openstack/setup.sh
+# patch StorageProfile with ReadWriteOnce Access
+k8s_patch_storage_profile
 
-. ./grant_permissions.sh
```

Comment:
so, can we drop that script from the repo then? :)

Reply:
dropps